webapps/frontend/ui/cockpit/client/scripts/directives/time-to-live/time-to-live-dialog.js

```diff
@@ -42,8 +42,42 @@ const Controller = [
     $scope.ttl = $scope.definition.historyTimeToLive;
 
     $scope.openBatchOperation = () => {
+      let batchSearchQuery = null;
+      const operation =
+        $scope.$parent?.resource === 'process-definition'
+          ? 'PROCESS_SET_REMOVAL_TIME'
+          : 'DECISION_SET_REMOVAL_TIME';
```

Comment:
❓Parent resource can only be these two?

🔧 If yes, then below you could use just an `if-else`. You could also move this assignment into the `if-else`, less comparisons and strings.